cri-o.yml

```diff
@@ -149,69 +147,75 @@
               make install && \
               make install.systemd && \
               make install.config
+
     - name: install policy.json in Ubuntu
-      shell: |
-              cd /root/src/github.com/kubernetes-incubator/cri-o && \
-              cp test/policy.json /etc/containers/policy.json
+      copy:
+        src: /root/src/github.com/kubernetes-incubator/cri-o/test/policy.json
+        dest: /etc/containers/policy.json
+        remote_src: true
+
     - name: build CNI stuff
       shell: |
               cd /root/src/github.com/containernetworking/plugins && \
               ./build.sh && \
               mkdir -p /opt/cni/bin && \
               cp bin/* /opt/cni/bin/
+
     - name: run CRI-O with systemd cgroup manager
       replace:
         regexp: 'cgroupfs'
         replace: 'systemd'
         name: /etc/crio/crio.conf
         backup: yes
-      when: (ansible_distribution == 'Fedora' or ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
+      when: ansible_distribution in ['Fedora', 'RedHat', 'CentOS']
+
     - name: run with overlay2
       replace:
         regexp: 'storage_driver = ""'
         replace: 'storage_driver = "overlay2"'
         name: /etc/crio/crio.conf
         backup: yes
+
     - name: add overlay2 storage opts on RHEL/CentOS
       lineinfile:
         dest: /etc/crio/crio.conf
         line: '"overlay2.override_kernel_check=1"'
         insertafter: 'storage_option = \['
         regexp: 'overlay2\.override_kernel_check=1'
         state: present
-      when: (ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS')
+      when: ansible_distribution in ['RedHat', 'CentOS']
+
     - name: enable and start CRI-O
       systemd:
         name: crio
         state: started
         enabled: yes
         daemon_reload: yes
+
     - name: modprobe br_netfilter
-      command: "modprobe br_netfilter"
+      modprobe: name=br_netfilter
       ignore_errors: true
+
     - name: tune sysctl
-      lineinfile:
-        line: "net/bridge/bridge-nf-call-iptables = 1"
-        dest: /etc/sysctl.conf
-        insertafter: 'EOF'
-        regexp: '\/net\/bridge\/bridge-nf-call-iptables = 1'
-        state: present
-      ignore_errors: true
-    - name: reload sysctl
-      command: "sysctl -p"
+      sysctl:
+        name: net/bridge/bridge-nf-call-iptables
+        value: 1
+        sysctl_file: /etc/sysctl.conf
```

Comment:
https://docs.ansible.com/ansible/sysctl_module.html can drop this as it's the default